### DISCUSSION TITLE
添加右键开关管理

### DISCUSSION THREAD

**Discussion by ilovelyyou** [immersive-translate/immersive-translate#406] [Ideas]
### 新功能 | New Feature

1.对自己想使用功能进行保留
2.去除使用率低的功能

### 为什么您想要添加这些功能 | Why you want to add these features

1.可以使浏览器更简洁
2.寻找自己想要功能更快捷

### 提议 | Proposal

可以在设置中进行分项设置，如
长按菜单管理
——————
翻译PDF        □（yes or not）
阅读本地       □    （yes or not）
电子书

**Comment by theowenyoung** [comment]
这个是什么浏览器？ 它似乎合并了 网页中的右键，和 浏览器图标里的右键的功能。
